dev/tasks/verify-rc/github.macos.arm64.yml

```
@@ -22,7 +22,7 @@
 jobs:
   verify:
     name: "Verify release candidate on macOS"
-    runs-on: {{ github_runner|default("self-hosted") }}
+    runs-on: {{ github_runner|default(["self-hosted", "macOS", "arm64"]) }}
```

```suggestion
    runs-on: {{ github_runner|default('["self-hosted", "macOS", "arm64"]') }}
```
Should this not be quoted to be inserted verbatim?

---

Looks like it isn't strictly necessary